src/core/product/index.js

```diff
@@ -1,11 +1,11 @@
 // @flow
 const { pubClient } = require('../client');
 
-// getSnapshot returns a Promise that checks the products current status
+// getProductSnapshot returns a Promise that checks the products current status
 // this seems to be set to BTC automatically.
-function getSnapshot(cc: string): Promise<number | Error> {
+function getProductSnapshot(currency: string): Promise<number | Error> {
```

Comment:
This is actually not just a 'currency', but a 'CoinCurrency'.

Examples are:

- 'LTC-USD'
- 'BTC-USD'
- 'ETH-USD'

and so on.

I used CC (CoinCurrency) to describe this. Does it make sense to use `coinCurrency` or another description for this? I am undecided and was not sure what to call it.